### DISCUSSION TITLE
Each island with its own world

### DISCUSSION THREAD

**Discussion by JakkitDE** [BG-Software-LLC/SuperiorSkyblock2#1961] [Pending Feature Requests]
We are leading a 10 years old skyblock server and we are using SuperiorSkyblock. For us its the best skyblock plugin available <3
My team mates have a feature request: PerWorldIslands.. So each Island has its own world..

**Comment by Tuogptuan** [comment]
SlimeWorldManager exists for a reason

**Comment by Lemvs** [comment]
@JakkitDE as Tuogptuan wrote, you can use SlimeWorlds and SlimeWorlds Module for SSB.
@OmerBenGera you can close it.
